src/core/qfieldcloudprojectsmodel.cpp

```diff
@@ -2472,9 +2472,9 @@ bool QFieldCloudProjectsFilterModel::filterAcceptsRow( int source_row, const QMo
       break;
   }
 
-  QString Name = mSourceModel->data( currentRowIndex, QFieldCloudProjectsModel::NameRole ).toString();
-  QString Description = mSourceModel->data( currentRowIndex, QFieldCloudProjectsModel::DescriptionRole ).toString();
-  QString Owner = mSourceModel->data( currentRowIndex, QFieldCloudProjectsModel::OwnerRole ).toString();
+  const QString Name = mSourceModel->data( currentRowIndex, QFieldCloudProjectsModel::NameRole ).toString();
```

Comment:
Should this be lowercase? `name`?

Reply:
I was reviewing the code and sensed that something was off, but I couldn't pinpoint exactly what it was. That's it lowercase +1.